doc/chips/10.rst

```diff
@@ -250,6 +250,78 @@ block while the rules for Phase 2 may be applied to the second, without a
 constant check to the location of the ``.init()`` call which is the linchpin for
 syntax 1.
 
+Child's fields could depend on parent's fields
```

Comment:
Also, note that the section titles are referenced in the outline portion at the head of the document, but it was not updated to include these new sections.